### PR TITLE
Initial PR for KPL Release 0.14.0

### PR DESCRIPTION
**Pull request by Cory-Bradshaw**
*Issue #, if available:*

*Description of changes:*
Initial PR for KPL Release 0.14.0

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,291 @@
...
+* Note: Windows will be unsupported going forward for this library.
+* [PR #260] Added endpoint for China Ningxia region (cn-northwest-1).
+* [PR #277] Changed mechanism to update the shard map
+  * Switched to using ListShards instead of DescribeStream, as this is a more scalable API
+  * Reduced the number of unnecessary shard map invalidations
+  * Reduced the number of unnecessary update shard map calls
+  * Reduced logging noise for aggregated records landing on an unexpected shard
+* [PR #276] Updated AWS SDK from 1.0.5 to 1.7.180
+* [PR #275] Improved the sample code to avoid need to edit code to run.
+* [PR #274] Updated bootstrap.sh to build all dependancies and pack binaries into the jar.
```
Comment (micah-jaffe) [inline CHANGELOG.md]: Typo: dependencies

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,291 @@
+## Changelog
+### 0.14.0
+
+* Note: Windows will be unsupported going forward for this library.
+* [PR #260] Added endpoint for China Ningxia region (cn-northwest-1).
```
Comment (micah-jaffe) [inline CHANGELOG.md:5]: Can we add hyperlinks to the PRs/issues?

**Review by micah-jaffe — APPROVED** [review]
Minor comments, but overall looks good to me.

File: README.md [hunk 3]
```diff
@@ -14,6 +17,20 @@ If you have further questions [please open a GitHub Issue](https://github.com/aw
 This is a restatement of the [notice published](https://docs.aws.amazon.com/streams/latest/dev/kinesis-kpl-upgrades.html) in the [Amazon Kinesis Data Streams Developer Guide][kinesis-developer-guide]
 
 ## Release Notes
```
Comment (sayantacC) [inline README.md:19]: Lets add a note pointing to the release notes in the changelog file.

Comment (sayantacC) [reply README.md:19]: I am fine with the rest of the changes.

**Pull request merged**